.github/workflows/build-dotnet.yml

```diff
@@ -3,6 +3,9 @@ name: Build .NET SDK
 on:
   workflow_dispatch:
   pull_request:
+    paths:
+      - "dotnet/**"
+      - "proto/**"
```

Comment:
Add one for the `build-python.yml` too please!

Reply:
Sure